### PR TITLE
Move confirm to substep path and remove index and install steps

### PR DESCRIPTION
**Pull request by lsl**
#### Changes proposed in this Pull Request

* Applies the same substep approach introduced in #58333 to the new stand alone confirm step from @retrofox in #58024
* Removes the install step (at least for now as it's not in use / working) - I'm expecting we replace the need for this by implementing all the plugin related installation in the transfer step itself using the new endpoint.
* Removes the now redundant Stepwrapper at the folder root
* Fixes a missing prop error in the confirm stepwrapper (missing stepName error shows up only when loading the screen on an already transferred site)

#### Testing instructions

* Confirm the transfer flow still works http://calypso.localhost:3000/start/woocommerce-install/confirm?siteSlug=wooptestupgrade.wordpres\s.com
* Confirm the normal start flow still works http://calypso.localhost:3000/start

Related to #

### CODE REVIEW

**Review by retrofox — APPROVED** [review]
LGTM. It works as expected.
However, this is something that took my attention:

> Removes the install step (at least for now as it's not in use / working) - I'm expecting we replace the need for this by implementing all the plugin related installation in the transfer step itself using the new endpoint.

Should we consider the case when the site is already an Atomic site? In that case, I think we'll still need this install step.

**Comment by retrofox** [comment]
resolving conflicts [here](https://github.com/Automattic/wp-calypso/pull/58441#event-5666916160), just FYI

File: client/signup/steps/woocommerce-install/confirm/index.tsx [hunk 1]
```diff
@@ -167,12 +153,11 @@ export default function Confirm( {
 			allowBackFirstStep={ true }
 			backUrl="/woocommerce-installation"
 			headerText={ headerTitle }
-			fallbackHeaderText={ headerTitle }
```
Comment (retrofox) [inline client/signup/steps/woocommerce-install/confirm/index.tsx]: removing this prop triggers the following error in the client

image

**Review by retrofox — CHANGES_REQUESTED** [review]
Sorry, requesting changes because of [this](https://github.com/Automattic/wp-calypso/pull/58441/files#r756154048).

**Review by retrofox — APPROVED** [review]
LGTM

File: client/signup/steps/woocommerce-install/confirm/index.tsx (hunk 1 above)
Comment (lsl) [reply client/signup/steps/woocommerce-install/confirm/index.tsx]: Why is the fallback header being sent in the prop headerText when we sent headerText?

**Comment by lsl** [comment]
Thanks for review/fixing/merging!

**Pull request merged**